src/Core/Algorithms/Field/RefineTetMeshLocallyAlgorithm.h

```diff
@@ -0,0 +1,98 @@
+/*
+   For more information, please see: http://software.sci.utah.edu
+
+   The MIT License
+
+   Copyright (c) 2015 Scientific Computing and Imaging Institute,
+   University of Utah.
+
+   
+   Permission is hereby granted, free of charge, to any person obtaining a
+   copy of this software and associated documentation files (the "Software"),
+   to deal in the Software without restriction, including without limitation
+   the rights to use, copy, modify, merge, publish, distribute, sublicense,
+   and/or sell copies of the Software, and to permit persons to whom the
+   Software is furnished to do so, subject to the following conditions:
+
+   The above copyright notice and this permission notice shall be included
+   in all copies or substantial portions of the Software.
+
+   THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+   OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+   FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+   THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+   FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+   DEALINGS IN THE SOFTWARE.
+*/
+
+#ifndef CORE_ALGORITHMS_FIELD_RefineTetMeshLocally_H
+#define CORE_ALGORITHMS_FIELD_RefineTetMeshLocally_H 1
+
+// Datatypes that the algorithm uses
+#include <Core/Datatypes/DatatypeFwd.h> 
+
+// Base class for algorithm
+#include <Core/Algorithms/Base/AlgorithmBase.h>
+
+// for Windows support
+#include <Core/Algorithms/Field/share.h>
```

Comment:
@moritzdannhauer Make sure share.h is the final include in the file. The Windows build is broken due to this.